src/main/pages/overview/infra-support.adoc

```diff
@@ -10,72 +10,62 @@ folder: overview
 [id="introduction"]
 == Introduction
 
-Eclipse Che runs on different kind of infrastructure and container engine:
+Eclipse Che runs on several infrastructures and container engines:
+
 
 * Docker
 * OpenShift: *https://www.openshift.com/container-platform/index.html[OpenShift Container Platform (OCP)]*, *https://www.openshift.com/features/index.html[OpenShift Online (OSO)]*, *https://access.redhat.com/products/openshift-dedicated-red-hat/[OpenShift Dedicated (OCD)]*, *https://www.openshift.org/minishift/[MiniShift]*
 * Kubernetes
 
-Each of the infrastructures is providing its own specificities which are detailed in this page.
 
 [id="comparison-table"]
-=== Comparison Table
+=== Available features for different infrastructures
```

Comment:
Seems like a formatting issue with the table. When I click 'View' above, the table renders like this: 
![screenshot from 2018-10-01 18-38-46](https://user-images.githubusercontent.com/16647625/46290384-528e4300-c5a9-11e8-82aa-6ccd44cd98a9.png)
Notice the empty last column.